pkg/cluster/cluster.go

```diff
@@ -102,22 +102,25 @@ func new(config Config, s *spec.ClusterSpec, stopC <-chan struct{}, wg *sync.Wai
 		status:  &Status{},
 	}
 
-	if err := c.prepareBackupAndRestore(); err != nil {
-		if err == errBackupUnsetRestoreSet {
-			c.logger.Errorf("%v. This is not allowed. Cluster to be dead.", err)
+	if isNewCluster {
```

Comment:
I don't think this is equivalent to "creating new cluster".
This var would better be named "IsControllerRefreshingInformation".

I'm mostly concerned with the case that the TPR object was created before etcd operator starts/restarts, and then etcd operator won't do anything.